interface-definitions/include/firewall/common-rule.xml.i

```diff
@@ -95,6 +95,51 @@
     </constraint>
   </properties>
 </leafNode>
+<node name="ct-status">
+  <properties>
+    <help>Connection status in conntrack</help>
+  </properties>
+  <children>
+    <leafNode name="dnat">
+      <properties>
+        <help>Set when connection needs DNAT in original direction</help>
```

Comment:
I think a better wording would be `Match connections that are subject to destination NAT`. The word "set" is clear in the original context in the nftables flag descriptions, but in our CLI I feel it would be ambiguous and may be interpreted as if this option _sets_ a flag, rather than make the rule match connections where that flag is _already set_.